packages/@okta/vuepress-site/docs/guides/terraform-enable-org-access/main/index.md

```diff
@@ -91,6 +91,19 @@ If the key isn't in the right format, convert it to the correct format using the
 
 > **Note:** The file that contains the converted private key must begin with `-----BEGIN RSA PRIVATE KEY-----`. If not, try step two again.
 
+## Assign admin roles
+
+Limit the access of the service app to your org by assigning it one or more admin roles. This increases the security of your org by defining which resources are accessible.
+
+1. In the Admin Console, open the service app and select **Admin roles**.
```

Comment:
Probably needs a step to get to the apps page, so something like this:

```suggestion
1. In the Admin Console, go to **Applications** > **Applications** to view the current app integrations
1. Select the service app, and then select **Admin roles**.
```

Reply:
This is an "I agree, but then it's inconsistent with the other instances of steps in the article". If you look at the other steps in this article you'll see a pattern. The first set in the Overview gives explicit directions starting with selecting Applications > Applications. The next set in that section refers to opening the service app "create above."

In the next section (Create access credentials), the first step is structured just like the one I added in the new section.

The implication is that the person is going through the whole article so doesn't need the extra step. If that's not the case, then if we make this change, we should also change the one in Create access credentials. I don't care which.

Do you still want the change?

Reply:
No. Let's stay consistent with the rest of the article. I was looking only at your changes so didn't read it in context.